#msg file which represent the final output of our structure. This message is the read by the UI component to update the map
float32 x
float32 y
string  place_name
